app/lib/actions/katello/content_view_version/import.rb

```diff
@@ -25,7 +25,8 @@ def plan(organization:, path:, metadata:, library: false)
 
           sequence do
             plan_action(AutoCreateProducts, organization: content_view.organization, metadata: metadata)
-            plan_action(AutoCreateRepositories, organization: content_view.organization, metadata: metadata)
+            #plan_action(AutoCreateRepositories, organization: content_view.organization, metadata: metadata)
```

Comment:
```suggestion
```

Reply:
when having both of them enabled i get a 409 conflict if i try to do an import again